backend/e4h-services/ingestion-service/app/utils/fieldplan_service_client.py

```
@@ -44,6 +47,42 @@ def create_fieldPlan_facility(self, request_info: RequestInfo, fieldPlan_id: str
+                {
+                    "facilityId": facility_id,
+                    "fieldPlanId": fieldPlan_id,
+                    "isdeleted": False,
+                    "tenantId": "in"
+                }
+                for facility_id in facility_ids
+            ]
+        }
+        logger.trace(f"Bulk creating field plan facilities: fieldplan_id={fieldPlan_id}, count={len(facility_ids)}")
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)